tests/scattering1d/test_numpy_scattering1d.py

```diff
@@ -29,6 +29,7 @@ def test_Scattering1D(self, backend):
         N = x.shape[-1]
         scattering = Scattering1D(J, N, Q, backend=backend, frontend='numpy')
         Sx = scattering(x)
+        assert Sx.shape == Sx0.shape
```

Comment:
Why this addition?

Reply:
because it can be really nice for debugging to have a separate `AssertionError` if the shapes are different than if the array contents are different